docs/maintainer/knowledge_base.md

```diff
@@ -498,6 +498,18 @@ requirements:
     - {{ pin_compatible("numpy") }}
 ```
 
+With MPI, openmpi is required for the build platform, but mpich is not required:
```

Comment:
What do you think if we add a link to one of the references in the issue? For example:

```suggestion
With MPI, [openmpi is required for the build platform](https://github.com/conda-forge/mpi4py-feedstock/blob/743d379c4a04dfbb9aa06191c02fefedb338330c/recipe/meta.yaml#L37), but mpich is not required:
```